deploy/strongswan.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
I didn't touch the first line from dns_myapi.sh, and I also see other files in dnsapi directory are having same line.
Do you really want me to use posix bourne shell instead of bash?